### PR TITLE
Fix of examples to match the 2.x version

### PR DESCRIPTION
**Pull request by pmk65**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks backward-compatibility?    | ❌
| Tests pass?   | ✔️
| Fixed issues  | #556
| Updated README/docs?   | ✔️
| Added CHANGELOG entry?   | ❌

**README.md update**
- Added links to new resources. (Autocomplete, Jodit, IMask)
- Removed references to outdated plugin configuration.
- Removed references to jQuery wrapper.

### CODE REVIEW

File: docs/selectize.html [hunk 1]
```diff
@@ -4,9 +4,9 @@
     <meta charset="utf-8" />
     <title>JSON Editor Selectize Integration Example</title>
     <script src="https://cdn.jsdelivr.net/npm/@json-editor/json-editor@latest/dist/jsoneditor.min.js"></script>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/selectize.js/0.12.1/js/standalone/selectize.js"></script>
-    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/selectize.js/0.12.1/css/selectize.bootstrap3.css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
```
Comment (schmunk42) [inline docs/selectize.html]: Was there a special reason for changing this?

Comment (pmk65) [reply docs/selectize.html]: I couldn't get it to work with the original libs.

Comment (schmunk42) [reply docs/selectize.html]: I meant using `https://` instead of `//`. That was the only change.

**Comment by schmunk42** [comment]
Thank you!

**Pull request merged**